generate-deployment-manifest.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+set -e
+
+template_prefix="shibboleth"
+STEMCELL_OS=${STEMCELL_OS:-ubuntu}
+
+infrastructure=$1
+
+fail() {
+	echo >&2 $*
+}
+
+if [[ "$infrastructure" != "aws-ec2" && "$infrastructure" != "warden" ]] ; then
+  fail "usage: ./make_manifest <aws-ec2|warden>"
```

Comment:
Yeah, no reason to have any of the dynamic generation bits in here..that's what the bosh release versions are for.. we know we'll use ec2, we know we'll have a BOSH director ID in secrets rather than grabbing from bosh status, so yeah, this generation script will need mostly gutted, and even the manifests reworked a bit from the bosh release.